### PR TITLE
Strip compression prefix from pubkey.

### PR DESCRIPTION
**Pull request by eko2one**
Fix for https://github.com/swentel/nostr-php/issues/10.

### CODE REVIEW

File: tests/GenerateTest.php [hunk 1]
```diff
@@ -13,7 +13,7 @@ public function testGetPublicKey()
     {
         $keys = new Keys();
         $private_key_hex = '67dea2ed018072d675f5415ecfaed7d2597555e202d85b3d65ea4e58d2d92ffa';
-        $public_key_hex = '027e7e9c42a91bfef19fa929e5fda1b72e0ebc1a4c1141673e2794234d86addf4e';
+        $public_key_hex  = '7e7e9c42a91bfef19fa929e5fda1b72e0ebc1a4c1141673e2794234d86addf4e';
```
Comment (swentel) [inline tests/GenerateTest.php:16]: Could you remove that addtional space before '=' ?
Or is that a code styling thing?

File: src/Keys.php [hunk 2]
```diff
@@ -39,7 +39,7 @@ public function getPublicKey($priv_hex)
         $priv = $ec->keyFromPrivate($priv_hex);
         $pub_hex = $priv->getPublic(true, 'hex');
 
-        return $pub_hex;
+        return substr($pub_hex, 2); // remove compression prefix 02 | 03
```
Comment (swentel) [inline src/Keys.php:42]: Hmm, more a fan of putting comments above the code - unless it's a specific code style ing

**Comment by swentel** [comment]
funky - left two tiny suggestions which are more code styling things. If those are ok, I'll merge!

File: src/Keys.php (hunk 2 above)
Comment (eko2one) [reply src/Keys.php:42]: :D No it is not a specific code styling, just too late at night

File: tests/GenerateTest.php (hunk 1 above)
Comment (eko2one) [reply tests/GenerateTest.php]: Done

**Comment by eko2one** [comment]
Ready to merge.

**Comment by swentel** [comment]
Hehe, don't forget to sleep ;-)

**Pull request merged**